backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/snapshots/CreateSnapshotForVmCommand.java

```diff
@@ -481,8 +481,7 @@ protected void endVmCommand() {
 
             return null;
         });
-        setSucceeded(getParameters().getTaskGroupSuccess() &&
-                (!getParameters().isLiveSnapshotRequired() || getParameters().isLiveSnapshotSucceeded()));
+        setSucceeded(true);
```

Comment:
based on your commit message, i actually see no reason to change this. if it failed before we will also fail here,  it's the overall command return status.
yes, it's nicer to just set `true`, but won't it be confusing? just a thought.

Reply:
this is the endAction's status, this just means that it executed successfully and there is no reason to retry (and the next line sets no retries anyway), this is harmless, but there will always be a message in the log stating that the endAction failed, which is incorrect